python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -1251,6 +1366,54 @@ def parallactic_angle(self, ra, dec, time=None):
         return pa_angle
 
     async def find_target(self, az, el, mag_limit, mag_range=2.0, radius=0.5):
+        """Make a cone search and return a target close to the specified
+        position.
+
+        Parameters
+        ----------
+        az: `float`
+            Azimuth (in degrees).
+        el: `float`
+            Elevation (in degrees).
+        mag_limit: `float`
+            Minimum (brightest) V-magnitude limit.
+        mag_range: `float`, optional
+            Magnitude range. The maximum/faintest limit is defined as
+            mag_limit+mag_range (default=2).
+        radius: `float`, optional
+            Radius of the cone search (default=2 degrees).
+
+        Returns
+        -------
+        target : `astropy.Table`
+            Target information.
+        """
+
+        target = None
+
+        if self.is_catalog_loaded():
+            self.log.debug("Searching internal catalog.")
+            try:
+                target = await self.find_target_local_catalog(
+                    az=az,
+                    el=el,
+                    mag_limit=mag_limit,
+                    mag_range=mag_range,
+                    radius=radius,
+                )
+            except RuntimeError:
+                self.log.info(
+                    "Could not find suitable target in local catalog. Continue and try with Simbad."
+                )
+
+        if target is None:
+            target = await self.find_target_simbad(
+                az=az, el=el, mag_limit=mag_limit, mag_range=mag_range, radius=radius
+            )
+
+        return target
+
+    async def find_target_simbad(self, az, el, mag_limit, mag_range=2.0, radius=0.5):
         """Make a cone search in Simbad and return a target close to the
         specified position.
```

Comment:
According to the search criteria, only targets from the HD catalog are returned but this is not reflected in the doc string. Please clarify that here.

Reply:
Why to you say only HD catalog starts are returned? The search criteria does not specify a catalog so targets from any catalog available in Simbad are returned.

Reply:
Line 1454 contains `"cat = HD"` and that's added to the search criteria for Simbad.

Reply:
oh wow! you are absolutely right.. I forgot I added that.. nice catch....